src/Components/Form/AddNewTodo.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+interface Props {
+  users: User[];
+  tasks: Todo[];
+  addTodo: (todo: Todo) => void,
+}
+
+interface State {
+  title: string;
+  userId: number;
+  isTitleEmpty: boolean;
+  isSelectUser: boolean;
+}
+
+export class AddNewTodo extends React.Component<Props, State> {
+  state: State = {
+    title: '',
+    userId: 0,
+    isTitleEmpty: false,
+    isSelectUser: false,
+  };
+
+  resetForm = () => {
+    this.setState({
+      title: '',
+      userId: 0,
+      isTitleEmpty: false,
+      isSelectUser: false,
+    });
+  };
+
+  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({
+      title: event.target.value,
+    });
+  };
+
+  handleSelectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    this.setState({
+      userId: +event.target.value,
+    });
+  };
+
+  getValueFromForm = (event: React.FormEvent) => {
+    event.preventDefault();
+    const { userId, title } = this.state;
+
+    if (title.trim().length === 0) {
+      this.setState(currentState => ({
+        isTitleEmpty: !currentState.isTitleEmpty,
+      }));
+
+      return;
+    }
+
+    if (userId === 0) {
+      this.setState(currentState => ({
+        isSelectUser: !currentState.isSelectUser,
+      }));
+
+      return;
+    }
+
+    const newTodo = {
+      userId,
+      id: this.props.tasks.length + 1,
```

Comment:
The mentor said that it's an incorrect way to assign a new Id to the property 'id'. Consider using array methods to find the element with the biggest id in the array or the math method to find the biggest integer.